debian/README

```diff
@@ -0,0 +1,84 @@
+# llmnrd - Link-Local Multicast Resolution Daemon
```

Comment:
Sure, it'd be nice to install it. But AFAIK neither a copy nor a symlink should be necessary (allthough my debian packaging skills are a bit rusty). Can't you just put README.md to debian/llmnrd-docs.docs and it will get copied from the package root? At least that's how we e.g. do it for the netsniff-ng deb package.